scripts/st2bootstrap-el8.sh

```diff
@@ -19,10 +19,17 @@ DEV_BUILD=''
 USERNAME=''
 PASSWORD=''
 ST2_PKG='st2'
-ST2MISTRAL_PKG='st2mistral'
 ST2WEB_PKG='st2web'
 ST2CHATOPS_PKG='st2chatops'
 
+check_for_rhel() {
+  if [[ $(cat /etc/os-release | grep 'ID="rhel"') ]]; then
+    RHEL=1
+  else
+    RHEL=0
+  fi
+}
```

Comment:
Nitpick comment.

Just noticed we're using it in `STEP='Check for RHEL' && check_for_rhel`, where "STEPS" are more about running important operations like installing st2 + dependencies, configuring them, etc.

How about construct like this, wouldn't it be easier?
```bash
# Check if the current OS is Red Hat Enterprise Linux
is_rhel() {
  return $(cat /etc/os-release | grep 'ID="rhel"')
}
```

And using it in code:
```bash
  # RabbitMQ on RHEL8 requires module(perl:5.26
  if is_rhel; then
    sudo yum -y module enable perl:5.26
  fi
```

Reply:
ok. I can refactor it like that.